src/pages/profile/ProfileSections/UserReviews.jsx

```diff
@@ -0,0 +1,19 @@
+import CommentCard from "../../../components/TypesOfRecipeCards/CommentCard";
+const UserReviews = ({ user, reviews }) => {
+  //Show only user's reviews
+  const isEmptyReviews = reviews.length === 0 ? true : false;
```

Comment:
```suggestion
  const hasReviews = reviews.length > 0;
```
no need to add `? true : false`, since it would already evaluate as boolean.
also, it's better to keep your boolean variable names "positive". for example, `isEmptyReviews` is negative and `!isEmptyReviews` is more confusing because you have to think in double negatives, whereas `!hasReviews` is more natural.